assets/scss/_tech-partner.scss

```diff
@@ -0,0 +1,238 @@
+.tech-partner__heading {
+  background: linear-gradient(114.84deg, #133E9C -22.37%, #00091E 63.82%);
+  .tech-partner__headline {
+    @media only screen and (max-width: 738px) {
+      text-align: center;
+    }
+    .tech-partner__headline-bg {
+      background-image: url(../img/internet-future.png);
+      background-repeat: no-repeat;
+      background-position: 100% 90px;
+      background-size: 650px;
+      .col-sm-8 {
+        @media only screen and (max-width: 600px) {
+          margin: 0 auto;
+        }
+      }
+      .tech-partner__cta-button-wrapper {
+        @media only screen and (max-width: 768px) {
+          text-align: left;
+        }
+        @media only screen and (max-width: 600px) {
+          text-align: center;
+        }
+      }
+      @media only screen and (max-width: 991px) {
+        background-position: 100% 240px;
+        background-size: 550px;
+      }
+      @media only screen and (max-width: 600px) {
+        background-position: 100% 480px;
+        background-size: 550px;
+        height: 850px;
+      }
+
+      @media only screen and (max-width: 500px) {
+        background-position: 100% 590px;
+        background-size: 350px;
+        height: 850px;
+      }
+   }
+   h1, p {
```

Comment:
Is it possible that we avoid modifying html tag directly?

Reply:
removed all tag references